buildSrc/build.gradle

```diff
@@ -44,7 +44,7 @@ dependencies {
   runtime "com.google.protobuf:protobuf-gradle-plugin:0.8.13"                                          // Enable proto code generation
   runtime "io.spring.gradle:propdeps-plugin:0.0.9.RELEASE"                                            // Enable provided and optional configurations
   runtime "com.commercehub.gradle.plugin:gradle-avro-plugin:0.11.0"                                   // Enable Avro code generation
-  runtime "com.diffplug.spotless:spotless-plugin-gradle:5.1.2"                                       // Enable a code formatting plugin
+  runtime "com.diffplug.spotless:spotless-plugin-gradle:5.6.0"                                       // Enable a code formatting plugin
```

Comment:
It looks like the spotless plugin version is also referenced in `plugins { }` above

Reply:
Sorry for the piecemeal review.. one more comment, it looks like there's a  5.6.1 bugfix release, can we go ahead and bump to that?

Reply:
Done! thanks for pointing this out.